site/sfguides/src/data_mapping_in_native_apps/data_mapping_in_native_apps.md

```diff
@@ -92,61 +124,43 @@ longitude DOUBLE,
 zip_code varchar(30),
 time_zone varchar(8)
 );
+
 --Create a file format for the file
 CREATE OR REPLACE FILE FORMAT LOCATION_CSV
 SKIP_HEADER = 1
-FIELD_OPTIONALLY_ENCLOSED_BY = '"'
+FIELD_OPTIONALLY_ENCLOSED_BY = '\"'
 COMPRESSION = AUTO;
+
 --create a stage so we can upload the file
 CREATE STAGE LOCATION_DATA_STAGE
-file_format = LOCATION_CSV;
-```
-
-Your database should now look like the following
-
-<img src="assets/producer_org.png" width="425" />
+file_format = LOCATION_CSV;"
 
+--Go ahead now and upload the file to the stage.  After the file is uploaded go back to
+your worksheet and copy the file into the table created earlier
```

Comment:
I suggest you split the `sh` block into two and have this as normal paragraph text instead of inside of the sh block (also `--` doesn't work as a comment for SH blocks)